management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java

```diff
@@ -91,9 +89,7 @@
 
 public class ClusteringManagementServiceTest {
```

Comment:
Want to have a chat about that test - but not related to the changes proposed in this PR.